app/assets/v2/js/pages/increase_bounty.js

```diff
@@ -0,0 +1,160 @@
+        }
+        if(isError){
+            unloading_button($(this));
+            return;
+        }
+        $(this).attr('disabled','disabled');
+
+        //setup web3
+        // TODO: web3 is using the web3.js file.  In the future we will move
+        // to the node.js package.  github.com/ethereum/web3.js
+        var isETH = tokenAddress == '0x0000000000000000000000000000000000000000';
+        var token_contract = web3.eth.contract(token_abi).at(tokenAddress);
+        var account = web3.eth.coinbase;
+        amount = amount * decimalDivisor;
+        // Create the bounty object.
+        // This function instantiates a contract from the existing deployed Standard Bounties Contract.
+        // bounty_abi is a giant object containing the different network options
+        // bounty_address() is a function that looks up the name of the network and returns the hash code
+        var bounty = web3.eth.contract(bounty_abi).at(bounty_address());
+
+        // setup inter page state
+        localStorage[issueURL] = JSON.stringify({
+            'timestamp': null,
+            'txid': null,
+        });
+
+        function web3Callback (error,result){
+            if(error){
+                mixpanel.track("Increase Bounty Error (funder)", {step: 'post_bounty', error: error});
+                console.error(error);
+                _alert({ message: "There was an error.  Please try again or contact support." });
+                unloading_button($('#submitBounty'));
+                return;
+            }
+
+            // update localStorage issuePackage
+            var issuePackage = JSON.parse(localStorage[issueURL]);
+            issuePackage['txid'] = result;
+            issuePackage['timestamp'] = timestamp();
+            localStorage[issueURL] = JSON.stringify(issuePackage);
+
+            _alert({ message: "Submission sent to web3." }, 'info');
+            setTimeout(function(){
+                mixpanel.track("Submit New Bounty Success", {});
+                document.location.href= "/funding/details/?url="+issueURL;
+            },1000);
+        }
+
+        var bountyAmount = parseInt($('input[name=valueInToken]').val(), 10); 
+        var ethAmount = isETH ? amount : 0
+        var bountyId = $('input[name=standardBountiesId]').val();
+        var fromAddress = $('input[name=bountyOwnerAddress]').val();
+
+        var errormsg = undefined;
+        if(bountyAmount == 0 || open == false){
+            errormsg = "No active funded issue found at this address. Are you sure this is an active funded issue?";
+        } 
+        if(fromAddress != web3.eth.coinbase){
+            errormsg = "Only the address that submitted this funded issue may increase the payout.";
+        }
+
+        if(errormsg){
+            _alert({ message: errormsg });
+            unloading_button($('#submitBounty'));
+            return;
+        }
+
+        function approveSuccessCallback () {
+            bounty.increasePayout(
```

Comment:
i think youll need to batch up, and call the `contribute()` method here too

Reply:
It worked during manual testing, but if you want I can do that. Out of curiosity: What makes you think that this is necessary?

Reply:
@mbeylin told me it was :P

Reply:
i believe per the standardbounties docs, those two methods complement each other https://github.com/ConsenSys/StandardBounties/

Reply:
I'll look into it.